docker-compose.yml

```diff
@@ -36,12 +38,29 @@ services:
       - SESSION_SECRET=${SESSION_SECRET:?required}
       - MEMCACHED_SECRET=${MEMCACHED_SECRET:?required}
 
+  migrations:
+    depends_on:
+      db:
+        condition: service_healthy
+    build:
+      args:
+        - TRANSCENDENCE_APP_DATA=${TRANSCENDENCE_APP_DATA:-/var/lib/transcendence-app}
+    environment:
+      - POSTGRES_HOST=db
+      - POSTGRES_PORT=5432
+      - POSTGRES_DB=ft_transcendence
+
   db:
     image: postgres:14.7-alpine3.17
     restart: always
     user: postgres
     environment:
       - POSTGRES_DB=ft_transcendence
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 5s
```

Comment:
how did you come up with these values for interval, timeout, retries?

Reply:
I first used the values from the [docs](https://docs.docker.com/compose/compose-file/#healthcheck), then decreased the interval and increased the retries because compose runs the first health check command after the interval has passed, increasing the time it takes to deploy.